include/rcutils/concat.h

```diff
@@ -1,42 +0,0 @@
-// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
why was this file removed?

I tried to find any mention but I couldn't.

Reply:
I just replaced all use of it with `rcutils_format_string()`, so it's unnecessary now.